### ISSUE TITLE
Black screen when starting

### ISSUE THREAD

**Issue by YukiXXL** [israpps/CheatDevicePS2#11]
**Describe the bug**
Black screen when starting the game/app with cheats enabled.

**To Reproduce**
1. Select the codes
2. Go to Main Menu
3. Start Game and select either the Disc or the app (ex. mc0:\BOOT\BOOT.ELF)

**Expected behavior**
Game/App starting normally.

**Environment (please complete as much of the following information as possible):**
 - PS2 Region: PAL
 - PS2 Model Type: FAT
 - CheatDevice version: Latest (Normal, exFAT, HDD)
 - Device CheatDevice is launched from: Memory Card / USB
 - Launch method: from uLe
 - System modifications: FreeMcBoot
- `CheatDevicePS2.ini` contents:
```
default ones
```

**Comment by dbedrenko** [comment]
> Device CheatDevice is launched from: Memory Card / USB

You haven't specified whether you launched from MC or USB.

> Start Game and select either the Disc or the app (ex. mc0:\BOOT\BOOT.ELF)

In what form is the game you're using? Is it a DVD? If so, is it an ESR patched burnt disc or a retail disc?

I had an ESR patched game and my mistake was to not select the `esr.elf` boot file when selecting "Start Game".

**Comment by YukiXXL** [comment]
> You haven't specified whether you launched from MC or USB.
Sorry. I wrote slash to say either MC or USB but issue is on both.

> In what form is the game you're using?
Same as above. Either disc or OPL, both crashes.

I tested with a friend and he has crashes with normal version while I have crashes with EXFAT version of the Cheat Device.

**Comment by YukiXXL** [comment]
Tested newest version. Got black screen when using normal version on Disc but works fine with OPL.
However, MMCE version works with Disc but produces black screen with OPL.

**Comment by Unixware** [comment]
mine worked ... once! : booted back menu (SD2PSX)  the OPL -> game launched.. the next day , just boots to ... black screen (crashed?), no matter I ve tried to launch that from USB/HDD .. same result , self-corrupted ( eg config) ?

**Comment by israpps** [comment]
remove the read/write database to check if it boots back

**Comment by Unixware** [comment]
thanks , alas no luck! I will try to recreate back the SD2PSX and re-install this app again

**Comment by Unixware** [comment]
deleting the r/w databse from the config does nothing! def this is not compatible with SD2PSX  (or that card is just rubbish)

at least I can instruct to directly load the OPL - so I am ok with that...

**Comment by israpps** [comment]
Does wLaunchELF isr with MMCE support browse the SDCard?

**Comment by Unixware** [comment]
yes but no worries, as I can run directly OPL , that is my target ... - it may be a glitch with that memcard anyway

**Comment by israpps** [comment]
> yes but no worries, as I can run directly OPL , that is my target ... - it may be a glitch with that memcard anyway

No, if wLaunchELF can read it, it's not a problem on the card. Cheat device is hanging somewhere else, most likely an RPC service 

Will see if I can reproduce
